tests/src/FunctionalJavascript/CopyClipboardTest.php

```diff
@@ -0,0 +1,68 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Drupal\Tests\oe_bootstrap_theme\FunctionalJavascript;
+
+use Drupal\FunctionalJavascriptTests\WebDriverTestBase;
+
+/**
+ * Tests the copy to clipboard functionality.
+ */
+class CopyClipboardTest extends WebDriverTestBase {
+
+  /**
+   * {@inheritdoc}
+   */
+  protected $defaultTheme = 'oe_bootstrap_theme';
+
+  /**
+   * {@inheritdoc}
+   */
+  protected static $modules = [
+    'oe_bootstrap_theme_helper',
+    'system',
+    'ui_patterns',
+    'ui_patterns_settings',
+    'ui_patterns_library',
+  ];
+
+  /**
+   * Tests the copy to clipboard behavior in the copyright overlay.
+   */
+  public function testCopyToClipboard(): void {
+    $assert_session = $this->assertSession();
+
+    // Create and log in an admin user with full permissions.
```

Comment:
comments we should add to add clarification where something is ambiguous or hard to understand.. where it brings extra value in short. These comments here and not needed.